awx/settings/development.py

```diff
@@ -179,3 +179,4 @@
     os.environ['SDB_NOTIFY_HOST'] = os.popen('ip route').read().split(' ')[2]
 
 WEBSOCKET_ORIGIN_WHITELIST = ['https://localhost:8043', 'https://localhost:3000']
+AWX_CALLBACK_PROFILE = True
```

Comment:
It's nice to just turn this on by default in the dev env so you can just `kill -SIGUSR1` on callback processes to profile them.